src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
 body {
+  font-size: 12px;
+  margin: 0;
+  background-color: gray;
+}
+
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.img {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.navigation {
+  list-style: none;
+  display: flex;
+  padding: 0;
   margin: 0;
 }
+
+.navbar {
+  display: inline-block;
+  position: relative;
+  text-align: center;
+  line-height: 60px;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+}
+
+.navigation li:not(:last-child) {
+  margin-right: 20px;
+}
+
+.navbar:hover {
+  color: #00acdc;
+}
+
+.navbar.is-active {
+  color: #00acdc;
+}
+
+.navbar.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background: #00acdc;
```

Comment:
The color #00acdc is used multiple times. The task requires using a CSS variable for the blue color. Consider defining a CSS variable for this color and using it throughout the stylesheet.